src/core/annotation.js

```diff
@@ -602,6 +602,14 @@ var WidgetAnnotation = (function WidgetAnnotationClosure() {
       this.setFlags(AnnotationFlag.HIDDEN);
     }
 
+    // Support ResetForm Action Widget
+    if (data.fieldType === 'Btn' && dict.get('A') !== undefined) {
+      var actionRef = dict.map.A.num;
+      var xrefObject = dict.xref.fetchIfRef(dict.map.A);
+      data.formAction = {};
```

Comment:
I can see that you're setting another property on `data`, but what I'm saying is that after *setting* the property, the property is not used elsewhere.

Taken in isolation, it seems that this extra property doesn't do anything. It is not documented anywhere as far as I can tell, and neither used by the pdf.js code (web/ and src/).